arviz/stats/stats.py

```diff
@@ -992,7 +992,7 @@ def summary(
     fmt: str = "wide",
     kind: str = "all",
     round_to=None,
-    include_circ=None,
+    circ_var_names=None,
```

Comment:
I think we should keep `include_circ` for now and raise a deprecation warning when used to ease transition